pkgs/top-level/all-packages.nix

```diff
@@ -32253,10 +32253,11 @@ with pkgs;
       })
     nix
     nixStable
+    nix_2_3
     nix_2_4
     nixUnstable;
 
-  nixStatic = pkgsStatic.nix;
+  nixStatic = pkgsStatic.nix_2_3;
```

Comment:
Is this required? nix 2.4 builds statically just fine here (aarch64-linux)

Reply:
I couldn't get them to build on `x86_64-linux`, could you try?